src/m_menu.c

```diff
@@ -76,6 +76,7 @@ int mouseSensitivity_horiz2; // [FG] strafe
 int mouseSensitivity_vert2; // [FG] look
 
 int showMessages;    // Show messages has default, 0 = off, 1 = on
+int hideMessages;
```

Comment:
Not too sure about this variable name, as it might get confusing given that we already have `showMessages`. Suggestions?

Reply:
`hide_message_type` maybe?

Reply:
Sounds good.